webdriver/tests/bidi/script/call_function/invalid.py

```diff
@@ -421,3 +421,12 @@ async def test_params_include_shadow_tree_invalid_value(bidi_session, top_contex
             serialization_options=SerializationOptions(include_shadow_tree="foo"),
             target=ContextTarget(top_context["context"]),
             await_promise=True)
+
+@pytest.mark.parametrize("user_activation", ["foo", 42, {}, []])
```

Comment:
Can you please check why adding this test causes a timeout for Chrome? wpt.fyi doesn't show any result.

Reply:
Do you have a link to the failure?

Reply:
Given that you have pushed to this branch the test are running again, and I cannot find a reference to the wpt.fyi results for an older commit. So lets wait until the tests finished.

Reply:
It passes locally with chromium-bidi, so I think it would be related to some WPT infrastructure.

Reply:
It finished now, do you still see the same failure? I am not sure where to look for it (I am seeing some infra failures) and the known chrome stability error that says that we approach the slow test limit (due to the number of tests).

Reply:
sorry, I have noticed some further inconsistencies in naming so I triggered a re-run again